utils/hct/hctstart.cmd

```diff
@@ -115,21 +115,23 @@ echo  hctstart [-x86 or -x64] [path-to-sources] [path-to-build]
 echo.
 goto :eof
 
-:findcmake
+:findcmake 
+call :ifexistaddpath "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
```

Comment:
Yes, but I'll install and test those before committing the changes. I've only tested this with Community so far.